docs/Pflichtenheft.tex

```diff
@@ -333,8 +338,12 @@ \subsection{Überwachungskonsole}
 \begin{enumerate}
   \item[NF110] IP-Adressen müssen gültige IPv4 Adressen oder Hostnamen sein. % Abkürzungsverzeichnis für Dinge wie IPv4?
   \item[NF120] Ports dürfen Werte im Bereich 1024 bis 61000 annehmen.
+  \item[NF] Standardmäßig haben die \glspl{OPC UA Client} die Ports 25565, 25566, 25567 und 25568.
```

Comment:
Ports für die Clients habe ich aus den NFs herausgenommen, da ohne konkrete Zahlen diese dort keinen Sinn mehr machen. Dafür habe ich die Bedeutung der leeren Felder in den FAs aufgenommen.